frontend/apps/app/app/api/chat/stream/route.ts

```diff
@@ -0,0 +1,125 @@
+import {
+  type AgentWorkflowParams,
+  createSupabaseRepositories,
+  deepModelingStream,
+  invokeDbAgentStream,
+} from '@liam-hq/agent'
+import { NextResponse } from 'next/server'
+import * as v from 'valibot'
+import { createClient } from '@/libs/db/server'
+
+// https://vercel.com/docs/functions/configuring-functions/duration#maximum-duration-for-different-runtimes
+export const maxDuration = 300
```

Comment:
Let's change it to 800 in production implementation.

```suggestion
export const maxDuration = 800
```